### DISCUSSION TITLE
Ship the official Firefox deb package

### DISCUSSION THREAD

**Discussion by CattailAle** [linuxmint/discussions#563] [Ideas]
Mozilla now has an official apt repository for Debian-based distributions.
https://support.mozilla.org/en-US/kb/install-firefox-linux#w_install-firefox-deb-package-for-debian-based-distributions
It now seems unnecessary for Mint to package its own Firefox. Switching to Mozilla's package would have some advantages:

- Timely updates. Mint's Firefox package is frequently late in security updates, often for multiple days.
- Less work for Mint developers.
- Less bandwidth usage on Mint servers and mirrors.

**Comment by ajgreeny** [comment]
To avoid the snap version of firefox I have been using the directly downloaded .tar.bz archive of firefox (and now thunderbird as well) in my Xubun5u installation.
I share the downloaded archive and their config folders of both with my Mint 22 in order to avoid having different application versions of either application which can cause problems.
Both applications work superbly that way so I suspect Mint devs could avoid all that extra work without too many problems though I accept they would have to integrate such use into the Mint systems to avoid users having to do it manually post installation.

**Comment by mtwebster** [comment]
We do need to be more timely with updates (that's an organizational issue, not a technical one), but we're not going to add the Mozilla repo to a default installation.

Linux Mint's version *is* the tarfile from Mozilla, but packaged into debs. We just need to do better getting updates to the servers.
